cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/system/StreamSystemMetadataWriter.java

```diff
@@ -30,6 +30,8 @@
  */
 public class StreamSystemMetadataWriter extends AbstractSystemMetadataWriter {
 
+  public static final String EXPLORE_TAG = "explore";
```

Comment:
The other usages of "explore" are in DatasetSystemMetadataWriter, and that class has its own const EXPLORE_TAG, should we combine these two constants?